16.0.Dockerfile

```
@@ -123,6 +123,8 @@ RUN build_deps=" \
         click-odoo-contrib \
         debugpy \
         pydevd-odoo \
+        redis \
+        dnsq \
         git+https://github.com/mailgun/flanker.git@v0.9.15#egg=flanker \
```

You could have done this instead:

```suggestion
        git+https://github.com/mailgun/flanker.git@v0.9.15#egg=flanker[validator] \
```

Thanks anyway for the quick fix!

---

#584 opened for switching to this better form (in case they change in the future).